db/glue.c

```diff
@@ -870,8 +870,57 @@ int cmp_context(struct ireq *iq, unsigned long long genid,
 /*        TRANSACTIONAL INDEX ROUTINES        */
 /*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 
+int ix_isnullk(void *db_table, void *key, int ixnum)
+{
+    struct dbtable *db = db_table;
+    struct schema *dbixschema;
+    int ifld;
+    if (!db || !key || ixnum < 0 || ixnum >= db->nix) {
+        /* fprintf(stderr,
```

Comment:
It'd be better to replace them with `logmsg(LOGMSG_DEBUG, ...)`. An example is [here](https://github.com/bloomberg/comdb2/blob/master/bdb/file.c#L2012).